src/dali/plugin/input/csv/nexo.py

```diff
@@ -0,0 +1,149 @@
+# locking & unlocking deposits are transfers within Nexo. They cannot lock funds from other wallets/accounts.
+_INTEREST = "Interest"
+_LOCKING_TERM_DEPOSIT = "LockingTermDeposit"
+_UNLOCKING_TERM_DEPOSIT = "UnlockingTermDeposit"
+_FIXED_TERM_INTEREST = "FixedTermInterest"
+_DEPOSIT = "Deposit"
+
+
+class InputPlugin(AbstractInputPlugin):
+    __NEXO: str = "Nexo"
+
+    __TRANSACTION_TYPE_INDEX = 1
+    __CURRENCY_INDEX: int = 2
+    __AMOUNT_INDEX: int = 3
+    __SPOT_PRICE_INDEX: int = 4
+
+    __TIMESTAMP_INDEX: int = 7
+
+    __DELIMITER = ","
+
+    def __init__(
+        self,
+        account_holder: str,
+        transaction_csv_file: str,
+        native_fiat: Optional[str] = None,
+    ) -> None:
+
+        super().__init__(account_holder=account_holder, native_fiat=native_fiat)
+        self.__transaction_csv_file: str = transaction_csv_file
+        self.__logger: logging.Logger = create_logger(f"{self.__NEXO}/{self.account_holder}")
+
+    def load(self) -> List[AbstractTransaction]:
+        result: List[AbstractTransaction] = []
+
+        with open(self.__transaction_csv_file, encoding="utf-8") as transaction_csv_file:
+            # read CSV with header and skip first row
+            lines = reader(transaction_csv_file)
+
+            # Skip header line
+            header = next(lines)
+            self.__logger.debug("Header: %s", header)
+
+            for line in lines:
+                raw_data: str = self.__DELIMITER.join(line)
+                self.__logger.debug("Transaction: %s", raw_data)
+
+                transaction_type: str = line[self.__TRANSACTION_TYPE_INDEX].strip()
+                currency: str = line[self.__CURRENCY_INDEX].strip()
+                amount = line[self.__AMOUNT_INDEX].strip()
+                # there is no timezone information in the CSV, so we assume UTC
+                timestamp_with_timezone = f"{line[self.__TIMESTAMP_INDEX].strip()} -00:00"
+
+                common_params = {
+                    # although there is a transaction id in the CSV, it is not a transaction hash shared across exchanges, so it is uselsss
+                    # https://github.com/eprbell/dali-rp2/pull/60#issuecomment-1201481064
+                    "unique_id": Keyword.UNKNOWN.value,
+                    "plugin": self.__NEXO,
+                    "raw_data": raw_data,
+                    "timestamp": timestamp_with_timezone,
+                    "asset": currency,
+                }
+
+                # nexo does give us the spot price, but it's often 0 if a subcent value
+                # if it is non-zero, we use it, otherwise we use unknown as the value
+                # the spot price contains $ char, so we remove it
+                raw_spot_price = RP2Decimal(re.sub(r"[^\d.]", "", line[self.__SPOT_PRICE_INDEX]))
+
+                if raw_spot_price.is_zero():
+                    spot_price = Keyword.UNKNOWN.value
+                else:
+                    spot_price = str(raw_spot_price)
```

Comment:
Prefer ternary operator:
```suggestion
                spot_price = str(raw_spot_price) if not raw_spot_price.is_zero() else Keyword.UNKNOWN.value
```